[2024-03-04T14:56:28.068+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T14:56:24.910721+00:00 [queued]>
[2024-03-04T14:56:28.074+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T14:56:24.910721+00:00 [queued]>
[2024-03-04T14:56:28.074+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T14:56:28.074+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-03-04T14:56:28.074+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T14:56:28.083+0000] {taskinstance.py:1300} INFO - Executing <Task(FileSensor): file_sensor_task> on 2024-03-04 14:56:24.910721+00:00
[2024-03-04T14:56:28.087+0000] {standard_task_runner.py:55} INFO - Started process 172 to run task
[2024-03-04T14:56:28.089+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'file_sensor_task', 'manual__2024-03-04T14:56:24.910721+00:00', '--job-id', '3322', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpaouoq7fd']
[2024-03-04T14:56:28.090+0000] {standard_task_runner.py:83} INFO - Job 3322: Subtask file_sensor_task
[2024-03-04T14:56:28.126+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T14:56:24.910721+00:00 [running]> on host c630d9eea737
[2024-03-04T14:56:28.167+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=file_sensor_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T14:56:24.910721+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T14:56:24.910721+00:00
[2024-03-04T14:56:28.172+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2024-03-04T14:56:28.172+0000] {filesystem.py:64} INFO - Poking for file /home/adriano/ed/APP/data/data_turbine.json
[2024-03-04T14:56:28.172+0000] {filesystem.py:69} INFO - Found File /home/adriano/ed/APP/data/data_turbine.json last modified: 20240304145626
[2024-03-04T14:56:28.172+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2024-03-04T14:56:28.177+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=file_sensor_task, execution_date=20240304T145624, start_date=20240304T145628, end_date=20240304T145628
[2024-03-04T14:56:28.220+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T14:56:28.243+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
